libr/core/panels.c

```diff
@@ -1762,25 +1764,25 @@ static int systemShellCb(void *user) {
 
 static int stringCb(void *user) {
 	RCore *core = (RCore *)user;
-	addCmdfPanel (core, "search string: ", "\"/ %s\"");
+	addCmdfPanel (core, "search string: ", "/ %s");
 	return 0;
 }
 
 static int ropCb(void *user) {
 	RCore *core = (RCore *)user;
-	addCmdfPanel (core, "rop grep: ", "\"/R %s\"");
+	addCmdfPanel (core, "rop grep: ", "/R %s");
```

Comment:
uhmf wasnt those quotes necessary? or addCmdfPanel does the quoting by itself? otherwise can be a prooblem if u ropsearch for rax;ret for example

Reply:
great review because i wasn't familiar with the command just yet, need to check it after some sleep :P

also in that case, i don't know why i need this, but i need to unescape the cmd for savelayout and loadsavedlayout, which are used to save the cmd and restore by json.

otherwise backslash bothers the command to be carried out  actually. 

still, r_str_unescape didnt work properly for that....